java/ql/lib/semmle/code/java/dataflow/internal/TaintTrackingUtil.qll

```diff
@@ -113,6 +113,13 @@ private module Cached {
     node.getEnclosingCallable().getDeclaringType() instanceof NonSecurityTestClass or
     node.asExpr() instanceof ValidatedVariableAccess
   }
+
+  /**
+   * Holds if `guard` should be a sanitizer guard in all global taint flow configurations
+   * but not in local taint.
+   */
+  cached
```

Comment:
Caching this seems overkill.

Reply:
Oops, I think I just copied what was going on for defaultTaintSanitizer and didn't engage my brain enough.